src/Illuminate/Foundation/Console/ViewCreateCommand.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Str;
+
+class ViewCreateCommand extends Command
```

Comment:
Laravel have a base `Illuminate\Console\GeneratorCommand` that you should extend here instead of just `Command`.